sdk/storage/azure-storage-file-share/tests/perfstress_tests/download_to_file.py

```diff
@@ -0,0 +1,37 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import tempfile
+
+from azure_devtools.perfstress_tests import get_random_bytes
+
+from ._test_base import _ShareTest
+
+
+class DownloadToFileTest(_ShareTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        file_name = "downloadtest"
+        self.sharefile_client = self.share_client.get_file_client(file_name)
+        self.async_sharefile_client = self.async_share_client.get_file_client(file_name)
+
+    async def global_setup(self):
+        await super().global_setup()
+        data = get_random_bytes(self.args.size)
+        await self.async_sharefile_client.upload_file(data)
+
+    def run_sync(self):
+        with tempfile.TemporaryFile() as fp:
```

Comment:
Is this creating a unique temp file every call?  If so, can one temp file per instance be re-used?

Reply:
Same question as blob, is this test useful if we are just consuming the data differently (write to file vs write to null)?

Reply:
Only useful in the sense that in T1 we had a dedicated API for downloading to a file, that did the file creation.
In that sense, creating a unique temp file per download call is probably a close approximation.
However if we feel there's no real need to test this comparative scenario I'm fine with removing it altogether.

Reply:
I'm fine keeping the test, we can always decide to not run it in automation if we think the scenario is no longer interesting.